main.py

```diff
@@ -1,7 +1,32 @@
+import asyncio
+import sys
+import msvcrt
+
 height = 40 #40 characters
```

Comment:
all variables outside of a function are global so let's also make width, height, etc all caps

Reply:
Changed global variables to all caps